app/components/job-infrastructure-notification.js

```diff
@@ -33,10 +33,10 @@ export default Component.extend({
 
   @equal('queue', 'builds.macstadium6') isMacStadium6: null,
 
-  @computed('queue', 'job.config,dist')
-  isPreciseEOL(queue, dist) {
+  @computed('queue', 'job.config,dist', 'job.config.language')
```

Comment:
ah there’s a `,` in `job.config,dist` that should probably be a `.`?

Reply:
:man_facepalming:

Thanks for spotting that one, @backspace!